### PR TITLE
Scroll to DOM element

### PR DESCRIPTION
**Pull request by cyberhazard**

### CODE REVIEW

**Comment by Stanko** [comment]
Hey @cyberhazard,
Thanks for the PR, I'll review it and we can merge it.

File: animated-scroll-to.js [hunk 1]
```diff
@@ -6,6 +6,11 @@
   function __ANIMATE_SCROLL_TO(desiredOffset) {
     var userOptions = arguments.length <= 1 || arguments[1] === undefined ? {} : arguments[1];
 
+    if(desiredOffset instanceof HTMLElement){
+      var scrollTop = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop;
```
Comment (Stanko) [inline animated-scroll-to.js]: Please add spaces before `(` and after `)`

It is enough to use `window.scrollY || document.documentElement.scrollTop`, `pageYOffset` is just an alias for `scrollY`.

File: index.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
       background: #FAFAFA;
     }
 
-    .Page--withText {
+    .Page--withText{
```
Comment (Stanko) [inline index.html]: Please add space before `{`

File: index.html [hunk 3]
```diff
@@ -86,6 +86,14 @@
       background:         linear-gradient(to bottom,  #fff 0%, #ddd 100%);
     }
 
+    .Page--withElement{
```
Comment (Stanko) [inline index.html]: Please add space before `{`

**Comment by cyberhazard** [comment]
done

**Comment by cyberhazard** [comment]
If we add selector, It seems to be a little not obvious, queryselector returns the first item :)

**Comment by Stanko** [comment]
You are right, thanks again!

I will release it to npm when I get back to the office.

**Pull request merged**